src/main/java/org/breedinginsight/brapi/v2/BrAPIGermplasmController.java

```diff
@@ -178,7 +178,7 @@ private void batchProcessGermplasm(List<BrAPIGermplasm> germplasmList, String pr
         Pattern programKeyPattern = Utilities.getRegexPatternMatchAllProgramKeysAnyAccession(programKey);
         germplasmList.parallelStream().forEach(germplasm -> {
             // Set dbId
-            germplasm.germplasmDbId(Utilities.getExternalReference(germplasm.getExternalReferences(), "breedinginsight.org")
+            germplasm.germplasmDbId(Utilities.getExternalReference(germplasm.getExternalReferences(), "breeding-insight.net")
```

Comment:
What was the reason for this change? Should we be using the value defined in application.yml `brapi.server.reference-source`